### PR TITLE
add signup modal to news articles

### PR DESCRIPTION
**Pull request by xtina-starr**
Addresses: https://artsyproduct.atlassian.net/browse/GROW-632

This implements a full-user sign-up modals on /news/ pages on desktop.

screen shot 2018-12-11 at 2 19 01 pm

### CODE REVIEW

**Comment by eessex** [comment]
Code looks good but we should test this.

**Comment by xtina-starr** [comment]
Yeah I was planning to add some tests. I wanted to get this up and running against the CI because of the errors I was seeing. I should have added a WIP flag.

File: src/desktop/apps/article/components/__tests__/InfiniteScrollNewsArticle.test.js [hunk 1]
```diff
@@ -67,10 +65,13 @@ describe("<InfiniteScrollNewsArticle />", () => {
       marginTop: "50px",
       isMobile: false,
     }
+
+    clock = sinon.useFakeTimers()
   })
 
   afterEach(() => {
     window.history.replaceState.reset()
+    clock.restore()
```
Comment (eessex) [inline src/desktop/apps/article/components/__tests__/InfiniteScrollNewsArticle.test.js]: ooh, was this the missing piece?

Comment (xtina-starr) [reply src/desktop/apps/article/components/__tests__/InfiniteScrollNewsArticle.test.js]: Yup!! `sinon.useFakeTimers()` overrides the global `setTimeout` function.

**Pull request merged**